search/meilisearch.py

```diff
@@ -402,17 +418,21 @@ def get_search_params(
 
 def get_filter_rules(
     rule_dict: dict[str, t.Any], exclude: bool = False, optional: bool = False
-) -> list[str]:
+) -> list[str | list[str]]:
     """
     Convert inclusion/exclusion rules.
     """
     rules = []
     for key, value in rule_dict.items():
         if isinstance(value, list):
-            for v in value:
-                rules.append(
-                    get_filter_rule(key, v, exclude=exclude, optional=optional)
-                )
+            key_rules = [
```

Comment:
@zemelten These changes are not needed for the the bug we are fixing in this PR. I added only the changes from line 110-124 above to the master and ran the tests, they are passing just fine.